src/GlobalHotKeys.js

```diff
@@ -30,10 +31,11 @@ class GlobalHotKeys extends Component {
     allowChanges: PropTypes.bool
   };
 
-  getChildContext() {
-    return {
-      globalHotKeysParentId: this._id
-    };
+  constructor(props) {
+    super(props);
+
+    this._id = KeyEventManager.getInstance().registerGlobalKeyMap(props.keyMap);
+    this._childContext = { globalHotKeysParentId: this._id };
```

Comment:
⬜️ Add a comment explaining the rationale behind maintaining a separate property to contain the entire object.